### PR TITLE
Danger Zone [Profile]

### PR DESCRIPTION
**Pull request by igorosip0v**
https://github.com/worldcoin/developer-portal/assets/89008845/0eedffda-d15b-4d3d-88fb-5acbbc08336f

### CODE REVIEW

File: web-app-router/tailwind.config.ts [hunk 1]
```diff
@@ -134,7 +135,7 @@ const config: Config = {
         32: "calc(32 * 1rem / 16)",
       },
 
-      width: {
+      spacing: {
```
Comment (andy-t-wang) [inline web-app-router/tailwind.config.ts:138]: Why are we changing this?

File: web-app-router/scenes/Portal/Profile/DangerZone/page/index.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import { DeleteAccountDialog } from "../DeleteAccountDialog";
+
+export const DangerZone = () => {
+  const [open, setOpen] = useState<boolean>(false);
+
+  return (
+    <SizingWrapper className="py-8">
+      <div className="grid gap-y-8">
+        <UserInfo color="pink" name="Lisa" email="lisa@toolsforhumanity.org" />
+
```
Comment (andy-t-wang) [inline web-app-router/scenes/Portal/Profile/DangerZone/page/index.tsx:20]: Why is this hard coded?

File: web-app-router/tailwind.config.ts (hunk 1 above)
Comment (igorosip0v) [reply web-app-router/tailwind.config.ts:138]: This is allows to use these values with other classes, for example, I was needed it with max-w today. It also applies to paddings, margins, etc. Just not to duplicate

File: web-app-router/scenes/Portal/Profile/DangerZone/page/index.tsx (hunk 2 above)
Comment (igorosip0v) [reply web-app-router/scenes/Portal/Profile/DangerZone/page/index.tsx:20]: If we need to apply logic already, then we need to merge #524 first

**Review by andy-t-wang — APPROVED** [review]

**Pull request merged**